hack/build-and-push.sh

```diff
@@ -64,10 +70,10 @@ fi
 
 # First, build the image, with the version info passed in.
 # Note that an image will *always* be built tagged with the GIT_TAG, so we know when it was built.
-docker build --build-arg COMMIT=${COMMIT} --build-arg TAG=${BINARY_TAG} \
-  			-t ${REGISTRY}/admission-server:${GIT_TAG} .
-docker push ${REGISTRY}/admission-server:${GIT_TAG}
+docker build --build-arg COMMIT=${COMMIT} --build-arg TAG=${BINARY_TAG} ARCH=${ARCH} \
```

Comment:
I think we need to switch to `docker buildx build --platforms={...}`.

As-is, we won't get a single multi-arch image

Reply:
Ok i will check👍

Reply:
@akankshakumari393 are you still able to work on this? Anything we can help with?